editor-extensions/vscode/package.json

```diff
@@ -1,138 +1,139 @@
+	"description": "Grain support for Visual Studio Code.",
+	"version": "0.7.0",
+	"publisher": "grain-lang",
+	"keywords": [
+		"grain",
+		"grain-lang"
+	],
+	"icon": "grain_shorthand_white.png",
+	"engines": {
+		"vscode": "^1.43.0"
+	},
+	"license": "MIT",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/grain-lang/grain-language-server"
+	},
+	"bugs": {
+		"url": "https://github.com/grain-lang/grain-language-server/issues"
+	},
+	"categories": [
+		"Programming Languages"
+	],
+	"activationEvents": [
+		"onLanguage:grain"
+	],
+	"main": "./client/out/extension",
+	"contributes": {
+		"languages": [
+			{
+				"id": "grain",
+				"aliases": [
+					"Grain",
+					"grain"
+				],
+				"extensions": [
+					".gr"
+				],
+				"configuration": "./language-configuration.json"
+			},
+			{
+				"id": "grain-type",
+				"aliases": [
+					"Grain Type"
+				],
+				"configuration": "./language-configuration.json"
+			}
+		],
+		"grammars": [
+			{
+				"language": "grain",
+				"scopeName": "source.grain",
+				"path": "./syntaxes/grain.json"
+			},
+			{
+				"language": "grain-type",
+				"scopeName": "source.grain-type",
+				"path": "./syntaxes/grain-type.json"
+			}
+		],
+		"configuration": {
+			"type": "object",
+			"title": "Grain Language Server configuration",
+			"properties": {
+				"grain_language_server.enableLSP": {
+					"scope": "window",
+					"type": "boolean",
+					"default": true,
+					"description": "Enable the language server"
+				},
+				"grain_language_server.enableStatementLenses": {
+					"scope": "window",
+					"type": "boolean",
+					"default": true,
+					"description": "Enable per-statement lenses"
+				},
+				"grain_language_server.maxNumberOfProblems": {
+					"scope": "resource",
+					"type": "number",
+					"default": 100,
+					"description": "Controls the maximum number of problems produced by the server."
+				},
+				"grain_language_server.trace": {
+					"scope": "window",
+					"type": "string",
+					"enum": [
+						"off",
+						"messages",
+						"verbose"
+					],
+					"default": "off",
+					"description": "Traces the communication between VS Code and the language server."
+				},
+				"grain_language_server.cliPath": {
+					"scope": "window",
+					"type": "string",
+					"default": "grain",
+					"description": "Path to the grain cli"
+				},
+				"grain_language_server.debounce": {
+					"scope": "window",
+					"type": "number",
+					"default": 1000,
+					"description": "Interval in milliseconds to debounce compiler call"
+				}
+			}
+		}
+	},
+	"scripts": {
+		"compile": "tsc -b && node syntaxes/generate-grain-type.js",
+		"watch": "tsc -b -w",
+		"postinstall": "cd client && npm install && cd ../server && npm install && cd ..",
+		"test": "sh ./scripts/e2e.sh",
+		"vscode:prepublish": "npm run compile",
+		"test-compile": "tsc -p ./",
+		"format": "prettier --write .",
+		"check-format": "prettier --check .",
+		"deploy": "vsce publish; ovsx publish"
```

Comment:
@phated Should this be `vsce publish && ovsx publish` so the CI fails if `vsce publish` fails?

Reply:
Doesn't `&&` cause issues on Windows?

Reply:
A quick Google search claims that `&&` should work fine on Windows. Have you seen issues with it?

Reply:
Thought it might be similar to this: https://github.com/grain-lang/grain-lang.org/issues/142 ?

Reply:
Looks like the issue with that is the use of `&` and `wait`, since you'd need to use different stuff.

Reply:
```suggestion
		"deploy": "vsce publish && ovsx publish"
```